docs/troubleshooting.md

```diff
@@ -59,49 +64,24 @@ The following is an example of a DAG that maps `http://kuard.local:80/` to the
 
 ![Sample DAG](./dag-img/kuard-dag.png "Sample DAG")
 
-
 ## Interrogate Contour's gRPC API
 
 Sometimes it's helpful to be able to interrogate Contour to find out exactly the data it is sending to Envoy.
 Contour ships with a `contour cli` subcommand which can be used for this purpose.
 
-Because Contour runs as a sidecar with Envoy, you must run `contour cli` commands _inside_ the pod.
-The easiest way do this is via `kubectl exec`
-```
+Because Contour secures its communications with Envoy using Secrets in the cluster, the easiest way is to run `contour cli` commands _inside_ the pod.
+Do this is via `kubectl exec`:
+
+```sh
 # Get one of the pods that matches the examples/daemonset
 CONTOUR_POD=$(kubectl -n heptio-contour get pod -l app=contour -o jsonpath='{.items[0].metadata.name}')
 # Do the port forward to that pod
-kubectl -n heptio-contour exec $CONTOUR_POD -c contour contour cli lds
+kubectl -n heptio-contour exec $CONTOUR_POD -c contour -- contour cli lds --cafile=/ca/cacert.pem --cert-file=/certs/tls.crt --key-file=/certs/tls.key
```

Comment:
this isn't so bad, we can live with it, especially as people can copy paste from here